gke/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: clip-search
+  name: clip-search
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: clip-search
+  template:
+    metadata:
+      labels:
+        app: clip-search
+    spec:
+      containers:
+      # image registry URI should be replaced once GitHub Action is set up
+      - image: gcr.io/titanium-vision-318200/search_service
+        name: search-service-p4sml
+        imagePullPolicy: Always
+        ports:
+          - containerPort: 8080
```

Comment:
This is the port where our API would be exposed to, right?

Reply:
`8080` port what our `server` listens to on a `Pod`. It is exposed to `Service` object, then the `Service` object maps `80` port from client to the `Pod`'s `8080` port.

Reply:
Thanks.